packages/snaps-utils/src/versions.test.ts

```diff
@@ -1,25 +1,64 @@
+import { assert } from '@metamask/utils';
 import { getSnapPrefix } from './snaps';
 import { SnapIdPrefixes } from './types';
 import {
+  assertIsSemVerRange,
+  assertIsSemVerVersion,
   DEFAULT_REQUESTED_SNAP_VERSION,
   getTargetVersion,
   gtVersion,
-  isValidSnapVersionRange,
-  resolveVersion,
+  isValidSemVerRange,
+  isValidSemVerVersion,
+  resolveVersionRange,
   satisfiesVersionRange,
+  SemVerRange,
+  SemVerVersion,
 } from './versions';
 
+function v(value: string[]): SemVerVersion[];
+function v(value: string): SemVerVersion;
+/**
+ * Utility to cast string to version.
+ *
+ * @param value - String to cast.
+ * @returns The same string with different type.
+ */
+function v(value: string | string[]): SemVerVersion | SemVerVersion[] {
+  return value as any;
+}
+
+/**
+ * Utility to cast string to range.
+ *
+ * @param value - String to cast.
+ * @returns The same string with different type.
+ */
+function r(value: string): SemVerRange {
+  return value as SemVerRange;
+}
```

Comment:
These function names are not descriptive at all. Even though this is a test, I would strongly prefer giving these a proper name.

Reply:
They're are as short as possible by design.

All they usage could be replaced by `r("1.2.3") -> "1.2.3" as SemVerRange`, but that's overly descriptive and I found it hard to follow tests because of that.

Reply:
We have long tried to remove function names that are this short as a matter of principle.

Reply:
They will be rejected by the next major version of our ESLint config: https://github.com/MetaMask/eslint-config/pull/200

Reply:
Removed in commit